### DISCUSSION TITLE
Images

### DISCUSSION THREAD

**Discussion by YeungKC** [YeungKC/Hakuba#14] [Post]
This is a image test post.

![Yoo-Hoo! Swing](https://user-images.githubusercontent.com/11473691/178079841-cdd7df78-73a2-42c2-a50c-2ae172b9f407.JPG)

![](https://github.com/hexojs/hexo-theme-unit-test/blob/master/source/assets/wallpaper-2572384.jpg?raw=true)

![Caption](https://github.com/hexojs/hexo-theme-unit-test/blob/master/source/assets/wallpaper-2311325.jpg?raw=true)

![](https://github.com/hexojs/hexo-theme-unit-test/blob/master/source/assets/wallpaper-878514.jpg?raw=true)

**Comment by Tougee** [comment]
- test `comment`
